src/Collection/Collection.router.spec.ts

```diff
@@ -171,9 +171,9 @@ describe('Collection router', () => {
               ok: false,
               data: [
                 {
-                  dataPath: '',
+                  instancePath: '',
```

Comment:
the latest version of `ajv` updated the error prop `dataPath` to `instancePath` and the error message to a `must` when a required param is missing.